code/controllers/subsystem/security_level.dm

```diff
@@ -56,6 +56,13 @@ SUBSYSTEM_DEF(security_level)
 		SSshuttle.emergency.modTimer(old_shuttle_call_time_mod)
 		SSshuttle.emergency.modTimer(selected_level.shuttle_call_time_mod)
 
+	//SKYRAT EDIT ADDITION START- In the event of an orange alert, change the airlock permissions
+	if (istype(current_security_level, /datum/security_level/orange))
+		enable_engineering_access()
+	else
+		revoke_engineering_access()
```

Comment:
moving off orange should both
- have a timer for engineers to react and move out of restricted areas
- explain that engineers are about to lose accesses

Reply:
It's an IC issue. The alert shouldn't be changed until engineering says the incident is resolved and they've left the area.

Reply:
Not leaving the area is only an IC issue when the game has properly communicated that they should.
Plus, you have a line for adding access, so at least have one that says 'Additional engineering accesses have been revoked' or something similar; the timer isnt necessary as emergency maint access doesnt have either

Reply:
EA doesn't have a timer, so EO shouldn't, but yes I'll add an announcement.